src/openforms/registrations/contrib/objects_api/tests/test_templatetags.py

```diff
@@ -119,3 +119,17 @@ def test_render_json_summary_no_submission(self):
         )
 
         self.assertEqual(rendered, "{}")
+
+
+class AsJsonTests(TestCase):
+    def test_render_as_geo_json(self):
```

Comment:
and what happens when the map component is optional and left blank? I think the default/empty value is an empty string. That should not crash the registration backend.